scss/blocks/tabs.scss

```diff
@@ -258,8 +263,92 @@
   margin-bottom: 0px;
 }
 
-.accordion-body > :nth-child(2) {
+.accordion-body > :nth-child(2),
+.tabs-as-columns .col-sm > :nth-child(2) {
   // 2nd child, because the first one is anchor
-  padding-top: 0px;
+  padding-top: 10px;
   margin-top: 0px;
-}
\ No newline at end of file
+}
+
+.dos {
+  padding-right: 50px;
+}
+
+.dos,
+.donts,
+ul li {
+  line-height: 1.5;
+  list-style: none;
+}
+
+.dos ul li::before {
+  content: '';
+  display: block;
+  background-image: $check-mark;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: left top;
+  height: 28px;
+  width: 28px;
+  position: absolute;
+  left: 0;
+}
+.dos h3::before {
+  content: '';
+  display: block;
+  background-image: $check-bordered;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: left top;
+  height: 30px;
+  width: 30px;
+  position: absolute;
+  left: 0;
+}
+.donts h3::before {
+  content: '';
+  display: block;
+  background-image: $cross-bordered;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 30px;
+  width: 30px;
+  position: absolute;
+  left: 0;
```

Comment:
we could optimise some of this css as there is a bunch of repetition? things that are the same for both blocks can live under a block like this
```
.dos h3::before, .donts h3::before {
    // styling shared across both
}
```
and then the things that differ can have their own blocks like you have above :)